nixos/tests/acme-issue-81842.nix

```diff
@@ -0,0 +1,61 @@
+# When nginx depends on a service that is slow to start up, requesting
```

Comment:
Can this be moved somewhere into the other acme tests?

I doubt the test will be run if it's so disconnected.

Also, please address the TODOs.

Reply:
Reason I moved it out is because the test is growing rather large... 

yeh the TODOs should be renamed to FUTUREWORK; and they apply to the existing `acme.nix` tests as well; as they're mixing test setup (making sure pebble is talked to instead of letsencrypt prod) and actual testing, whilsts this was split up in the past.

Reply:
done